tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java

```diff
@@ -41,7 +43,10 @@
   private static final int MIN_TASKS = 3;
   /** The maximum number of threads for each task type. */
   private static final int MAX_TASKS = 6;
+  /** the maximum repeatable times in one task of size in {@link TestInfo}. */
+  private static final int MAX_REPEAT_TIMES = 6;
 
+  private CyclicBarrier mBarrier;
```

Comment:
right, that's what I meant here. and we could pass the barrier to the test in the constructor of each task